examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1815 @@
+{
```

Comment:
<p>Great plot, and amazing convergence and sampling with the DM model!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<blockquote> Great plot</blockquote><p>Thanks! Just fancy-fied it even more with KDEs of the posterior samples and legends.</p><p> </p><p>Thoughts? Too busy? Should the discussion/visualization of biased parameter estimation be split out entirely to a forest plot?</p>

Reply:
<p>I think, as you said, it's now better to remove the posterior and true. The forest plots illustrate it perfectly, and this one looks a bit too busy with everything in it. The new legend works nicely btw.</p>

Reply:
<p>I wrote that above: I think the posterior can be removed but the true values don't hurt as those are just lines</p>